webpack.dev.js

```diff
@@ -1,14 +1,26 @@
-const merge = require('webpack-merge');
-const common = require('./webpack.common');
-const path = require('path');
-const apiMocker = require('connect-api-mocker');
-
-module.exports = merge(common, {
-    mode: 'development',
-    devtool: 'source-map',
-    devServer: {
-        contentBase: path.join(__dirname, 'dist'),
-        port: 5000
-    }
-});
+const merge = require('webpack-merge');
+const common = require('./webpack.common');
+const path = require('path');
+const apiMocker = require('connect-api-mocker');
+
+module.exports = merge(common, {
+    mode: 'development',
+    devtool: 'source-map',
+    devServer: {
+        contentBase: path.join(__dirname, 'dist'),
+        port: 5000,
+
+        // Will serve index.html, if the file requested doesn't exist.
```

Comment:
Improved dev experience, by getting the dev server to:
- hit the server (which needs to run separately btw)
- return with the app bundle etc for routes that don't do anything else